airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json

```diff
@@ -94,9 +94,9 @@
             }
           },
           {
-            "title": "Standard Inserts",
```

Comment:
could you change the order of the options so this is the first in the list? this way it would look like this:
<img width="585" alt="Screen Shot 2022-02-01 at 2 32 40 PM" src="https://user-images.githubusercontent.com/6246757/152062755-9b00d281-e40b-48a3-b940-12583cafbd18.png">

screenshot obtained using instructions [here](https://docs.airbyte.com/connector-development/connector-specification-reference#demoing-your-specification)